### DISCUSSION TITLE
It does not appear to be charging

### DISCUSSION THREAD

**Discussion by JimWright4089** [iRobotEducation/create3_docs#46] [Questions] [answered]
I just received my Create 3.  I plugged the charger in and set the robot on it.  The green LED on the charger is blinking and the yellow LED on the interface board is blinking.  The LED ring is not on, and I can't seam to turn it on.

What do I need to do to get it charging?

-Jim

**Comment by JimWright4089** [comment]
I see on this page: https://iroboteducation.github.io/create3_docs/hw/electrical/

If the battery self-protects, its internal management system may refuse to charge until it is reset. Resetting the battery is accomplished by removing the battery from the robot for at least fifteen minutes, at which point it should be reinstalled in the robot and the robot placed on the charger.

How do I remove the battery?

**Reply by shamlian** [reply]
If you haven't modified your robot, there should be no need to remove and reset the battery out of the box; the robot's charging dock should not blink. Please e-mail us at [create@irobot.com](mailto:create@irobot.com) so we can try to help figure out what's wrong. To answer your second question, though, accessing the battery is possible by unscrewing the five captive screws marked with red asterisks in [this diagram](https://iroboteducation.github.io/create3_docs/hw/mechanical/#bottom-view).

**Comment by shamlian** [comment, marked as answer]
If you haven't modified your robot, there should be no need to remove and reset the battery out of the box; the robot's charging dock should not blink. Please e-mail us at [create@irobot.com](mailto:create@irobot.com) so we can try to help figure out what's wrong.

**Comment by JimWright4089** [comment]
I decided that since I have to wait, I might as well put it on the docking station.  It blinked green and the interface board blink yellow for an hour-ish, then all of a sudden the ring LED went white and the happy noise eventually sounded.  My Create 3 is now happy.

**Reply by shamlian** [reply]
Just a clarifying question -- did you mean your Create is _now_ happy? Because if it's not happy, we haven't received an e-mail, and we'd like to try to help.

**Reply by JimWright4089** [reply]
I thought it was happy but found out today it is not.  I'm sending an e-mail.
